data/resources/plugins/renameitem.py

```
@@ -61,7 +67,49 @@ def callback(self, *args):
+        window = self.app.get_widget('window')
+        title = "Rename document"
+        dialog = srvdlg.create(enable_response=True, dtype=dtype, title=title, body=text, widget=rename_widget)
+        dialog.add_response("preview", _("Preview"))
+        self.app.add_widget('dialog-rename', dialog)
+        dialog.connect('response', self._on_rename_response, rename_widget)
+        dialog.present(window)
+
+    def _on_rename_response(self, dialog, response, rename_widget):
+        if response == 'apply':
+            srvdlg = self.app.get_service('dialogs')
+            body = _(f"\nYou are about to rename this document")
```

Ruff (0.8.2)

Undefined name `_`

(F821)